netsim/ansible/templates/initial/linux/debian.j2

```diff
@@ -0,0 +1,152 @@
+echo -n 'Starting initial config ' && date
```

Comment:
So you just copied the "ubuntu.j2" file, added "get netplan" and removed a few things. So when we figure out something needs to be changed and we fix it in one of the files everyone knows it has to be fixed in the other file as well, right?

There's a reason Jinja2 has "include" functionality.